python/ray/autoscaler/_private/aws/node_provider.py

```diff
@@ -648,6 +649,23 @@ def fillout_available_node_types_resources(
                     autodetected_resources.update(
                         {"GPU": gpus[0]["Count"], f"accelerator_type:{gpu_name}": 1}
                     )
+                # TODO: AWS SDK (public API) doesn't yet expose the NeuronCore
+                #  information, work-in-progress.
```

Comment:
To help the reader understand when this work is actually ready to be done, can we reference an AWS docs page URL here that will be updated when this API is release? Can we also open an issue with the `amz` label to make we address this after this PR is merged?

Reply:
Created issue, https://github.com/ray-project/ray/issues/38473
Will update the TODO in the next revision.